werkzeug/wrappers.py

```diff
@@ -903,10 +912,16 @@ def get_data(self, as_text=False):
 
```

Comment:
Please add a `cache=True` argument here and also update the docstring accordingly.